.eslintrc

```diff
@@ -13,17 +16,19 @@
     }
   },
   "env": {
-    "es6": true,
-    "node": true,
```

Comment:
Is removal of `node` from `env` intentional`? I guess we should not need it as we develop clientside apps so we might as well remove it :+1:.